add documentation for assignCustomerToGuestCart mutation (#9252)
magento/devdocs: src/guides/v2.4/graphql/mutations/merge-carts.md

The mutation retains any items that were already in the logged-in customer's cart. If both the guest and customer carts contain the same item, `mergeCarts` adds the quantities. Upon success, the mutation deletes the original guest cart.

> There is a similar [assignCustomerToGuestCart]({{page.baseurl}}/graphql/mutations/assign-customer-to-guest-cart.html) mutation which works in a different way: it assigns a logged in customer and merge customer's shopping cart items to the guest shopping cart.

Reviewer: Add a `{:.bs-callout-info}` line before this line.

```suggestion
Use the [assignCustomerToGuestCart]({{page.baseurl}}/graphql/mutations/assign-customer-to-guest-cart.html) mutation to assign the contents of a logged-in customer's cart to a guest cart.
```

Author: done
